src/zodios.types.ts

```diff
@@ -567,6 +567,10 @@ export type ZodiosOptions = {
    * Should zodios validate parameters and response? Default: true
    */
   validate?: boolean | "request" | "response" | "all" | "none";
+  /**
+   * Use raw values from request/response rather than parsed values. Default: true
+   */
+  rawValues?: boolean | "request" | "response" | "all" | "none";
```

Comment:
all these options don't make real sense when combined with validate, i propose to simplify with the following:
```suggestion
  /**
   * Should zodios transform the request and response ? Default: true
   */
  transform?: boolean;
```

Reply:
I'd like to make it able to choose whether to transform `"response"`, `"request"` respectively? Would you mind it to be?

And it surely can be simplify to `"request" | "response" | boolean`.

Reply:
yes, go ahead.